changelog/fragments/spec-scoreacrd-fix.yaml

```
@@ -0,0 +1,19 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      There will be error thrown when there is spec is not part of Custom Resource. In many valid cases,
+      there will not be spec available in the CR. For valid cases as well, it was throwing an error.
+      Modified the code to throw an suggestion for the same.
```

The wording here is a little confusing to me. Something like this may be a bit less confusing:
```suggestion
      (scorecard): Fixes a bug where an error would be thrown if a Custom Resource did not have a `.spec` field, which is a perfectly valid scenario.  Instead of throwing an error scorecard will now issue a warning.
```

---

Updated.